README.md

```diff
@@ -16,12 +16,12 @@
 
 ## Quick start
 
-`npm start` will launch the app, configured to connect to our Rinkeby deployment.
+Install with `yarn` and launch the app with `yarn start`. By default, the app is configured to connect to the Ethereum Rinkeby testnet.
 
 For connecting to other chains / deployments, a few useful npm scripts are provided:
 
-- Mainnet: `npm run start:mainnet` will launch the app, configured to connect to our mainnet deployment
-- Local development: `npm run start:local` will launch the app, configured to connect to our [aragen](https://github.com/aragon/aragen) local development environment. It will also use the local IPFS daemon, if it detects one exists. If you're using the [aragonCLI](http://github.com/aragon/aragon-cli), you'll want to run this to connect to its local chain.
+- Ethereum Mainnet: `yarn run start:mainnet` will launch the app, configured to connect to the Ethereum mainnet
+- Local development: `yarn run start:local` will launch the app, configured to connect to our [aragen](https://github.com/aragon/aragen) local development environment. It will also use the local IPFS daemon, if it detects one exists. If you're using the [aragonCLI](http://github.com/aragon/aragon-cli), you'll want to run this to connect to its local chain.
```

Comment:
We don't _really_ have to include `run`, so we might as well remove it for simplicity.

Reply:
Fixed in 709b6b4.